src/EbmlElement.cpp

```diff
@@ -714,14 +714,33 @@ filepos_t EbmlElement::OverwriteHead(IOCallback & output, bool bKeepPosition)
   return Result;
 }
 
+filepos_t EbmlElement::OverwriteData(IOCallback & output, bool bKeepPosition)
+{
+	if (ElementPosition == 0) {
```

Comment:
Please follow the coding style the rest of the files use: indentation 2 spaces, no tabs.

Reply:
Sorry I missed this!  I fixed the indentation in the header but forgot to fix it here.